src/main/java/frc/robot/OperatorBoard.java

```diff
@@ -0,0 +1,119 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.button.Button;
+
+/**
+ * A wrapper class for our custom-made operator board. Does not have a
+ * constructor. Access statically. Access Buttons via public static state.
+ * Access axis values via methods. Use methods to make buttons light up.
+ */
+public class OperatorBoard {
+
+    private static Joystick HARDWARE;
+    public OperatorBoardButton BUTTON_ONE;
+    public OperatorBoardButton BUTTON_TWO;
+    public OperatorBoardButton BUTTON_THREE;
+    public OperatorBoardButton BUTTON_SIX;
+    public OperatorBoardButton BUTTON_ELEVEN;
+    public OperatorBoardButton BUTTON_TWELVE;
+    public OperatorBoardButton BUTTON_THIRTEEN;
+    public OperatorBoardButton BUTTON_FOURTEEN;
+
+    private static OperatorBoard instance;
+
+    /**
+     * The constructor is kept private so that I can make sure it is only run once,
+     * and that there is only ever one instance of OperatorBoard in existence.
+     */
+    public static OperatorBoard getInstance() {
+        if (instance == null) {
+            instance = new OperatorBoard();
+        }
+        return instance;
+    }
+
+    /**
+     * The constructor is kept private so that I can make sure it is only run once,
+     * and that there is only ever one instance of OperatorBoard in existence.
+     */
+    private OperatorBoard() {
+        HARDWARE = new Joystick(1);
+        BUTTON_ONE = new OperatorBoardButton(1);
+        BUTTON_TWO = new OperatorBoardButton(2);
+        BUTTON_THREE = new OperatorBoardButton(3);
+        BUTTON_SIX = new OperatorBoardButton(6);
+        BUTTON_ELEVEN = new OperatorBoardButton(11);
+        BUTTON_TWELVE = new OperatorBoardButton(12);
+        BUTTON_THIRTEEN = new OperatorBoardButton(13);
+        BUTTON_FOURTEEN = new OperatorBoardButton(14);
```

Comment:
why no constants?

Reply:
Thanks for the reminder!  In the control branch, I'll add constants.  For now, there's no implementation and no button assignment.